ReachabilityUI/ReachabilityUI/Shared/ReachabilityRepository.swift

```diff
@@ -32,19 +32,37 @@ final class ReachabilityManager: ReachabilityRepository {
         }
     }
     
+    private var isCellular = false {
+        didSet {
+            guard reachabilityDelegate != nil else { return }
```

Comment:
do you need this `guard`? Is it mean to be a performance optimization so you only check `isConnected` if `reachabilityDelegate` is actually something? If so...fine, nothing further 😄

Reply:
I added it, because it was there already in the "isConnected" property, but I think you're right, it's not needed, as far as I can see. It's being set from the ReachabilityUIManager's init function. I'm not sure, but maybe a network change can happen in-between these two lines:
```
public init() {
        reachabilityManager = ReachabilityManager()
        reachabilityManager.setup(self)
    }
```
Not sure if that would cause a crash, if the guard is not there?